### PR TITLE
Completed string-manipulation-1 exercise.

### PR DESCRIPTION
**Pull request by John-Baik**
string manip

### CODE REVIEW

File: string-manipulation-1/src/capitalize.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* exported capitalize */
+function capitalize(word) {
+  var i = 0;
+  var e = '';
```
Comment (TimelikeClosure) [inline string-manipulation-1/src/capitalize.js:4]: Please use more-descriptive variable names than `e`.

File: string-manipulation-1/src/capitalize.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      e += letter;
+      i++;
+    } else {
+      letter = word[i].toLowerCase();
+      e += letter;
+      i++;
+    }
+  }
+  return e;
+}
```
Comment (TimelikeClosure) [inline string-manipulation-1/src/capitalize.js:18]: Where's your pseudo-code for this function?

File: string-manipulation-1/src/get-last-char.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/* exported getLastChar */
+function getLastChar(string) {
+  var lastIndex = string.length - 1;
+  var lastChar = string[lastIndex];
+  return lastChar;
+}
+
+// define function getLastChar with one parameter string
+// define variable lastIndex that finds the last index of string
+// define variable lastChar that finds the last character of string by using lastIndex variable
```
Comment (TimelikeClosure) [inline string-manipulation-1/src/get-last-char.js:10]: How does `lastChar` find the last character using `lastIndex`?

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**